src/index.ts

```diff
@@ -1,87 +1,80 @@
-import {API, APIClient, APIClientOptions, FetchProvider, UInt128} from '@wharfkit/antelope'
-import BN from 'bn.js'
+import {APIClient, Asset, Name, NameType} from '@wharfkit/antelope'
+import * as SystemTokenContract from './contracts/system.token'
 
-import {PowerUpAPI} from './powerup'
-import {RAMAPI} from './ram'
-import {REXAPI} from './rex'
-
-export * from './powerup'
-export * from './ram'
-export * from './rex'
-
-interface ResourcesOptions extends APIClientOptions {
-    api?: APIClient
-    sampleAccount?: string
-    symbol?: string
-    url?: string
-}
-
-export interface SampleUsage {
-    account: API.v1.AccountObject
-    cpu: UInt128
-    net: UInt128
+interface TokenOptions {
+    apiClient: APIClient
+    systemTokenSymbol?: Asset.SymbolType
 }
 
-export const BNPrecision = new BN(1000 * 1000)
+export class Token {
+    readonly systemTokenContract: SystemTokenContract.Contract
```

Comment:
This isn't just for the system token, so all references like this shouldn't say `systemToken`.

For this (and everything like it) it should probably just be `Contract`. 

It also should reference the `Contract` instance from the contract kit and not this specific contract.

Reply:
I got a type error when I tried to reference the `Contract` class from the contract kit 🤔